vmck/settings.py

```diff
@@ -2,7 +2,11 @@
 import sentry_sdk
 from .utils import is_true
 from .base_settings import *  # noqa
-from .base_settings import QEMU_CPU_MHZ, SSH_USERNAME, base_dir
+from .base_settings import QEMU_CPU_MHZ
```

Comment:
Why keep the star import above this if you want to import all of these one by one?